invenio_rdm_records/resources/config.py

```diff
@@ -28,17 +28,29 @@
     "self": ItemLink(template='/api/records/{pid_value}'),
     "self_html": ItemLink(template='/records/{pid_value}'),
     "files": ItemLink(template='/api/records/{pid_value}/files'),
+    "versions": ItemLink(template='/api/records/{pid_value}/versions'),
+    "latest": ItemLink(template='/api/records/{pid_value}/versions/latest'),
+    "latest_html": ItemLink(template='/records/{pid_value}/versions/latest'),
+    "access_links": ItemLink(
+        template='/api/records/{pid_value}/access/links', permission="manage"
+    ),
 })
 
 
 DraftLinks = LinksSchema.create(links={
     "self": ItemLink(template="/api/records/{pid_value}/draft"),
     "self_html": ItemLink(template="/uploads/{pid_value}"),
     "files": ItemLink(template="/api/records/{pid_value}/draft/files"),
+    "versions": ItemLink(template='/api/records/{pid_value}/versions'),
+    "latest": ItemLink(template='/api/records/{pid_value}/versions/latest'),
+    "latest_html": ItemLink(template='/records/{pid_value}/versions/latest'),
```

Comment:
These might point to something not existing right? Is that ok?

Reply:
yes indeed
right now, those links would simply 404, which isn't *entirely* wrong, but it'd probably be saner to only include them if it wouldn't lead to a 404 (more effort to make this conditional, of course), or simply remove them

Reply:
We have this: https://github.com/inveniosoftware/marshmallow-utils/blob/master/marshmallow_utils/fields/links.py#L52 for conditional rendering.

But it's probably fine for now...